src/main/java/org/jmxtrans/agent/StatsDOutputWriter.java

```diff
@@ -148,40 +140,56 @@ public synchronized void writeQueryResult(String metricName, String metricType,
         String type = "gauge".equalsIgnoreCase(metricType) || "g".equalsIgnoreCase(metricType) ? "g" : "c";
         if (statsType.equals(STATSD_DATADOG)) {
             sb.append(metricNamePrefix)
-                    .append(".")
```

Comment:
could you please revert the formatting changes here and elsewhere to make the changes more clear.